cmd/juju/common/controller_test.go

```diff
@@ -116,7 +116,7 @@ func (s *controllerSuite) TestWaitForAgentAPIReadyWaitsForSpaceDiscovery(c *gc.C
 	s.mockBlockClient.discoveringSpacesError = 2
 
 	runInCommand(c, func(ctx *cmd.Context, base *modelcmd.ModelCommandBase) {
-		err := WaitForAgentInitialisation(ctx, base, "controller", "default")
+		err := WaitForAgentInitialisation(ctx, base, false, "controller", "default")
```

Comment:
the only difference for `caas` and `iaas` is the Infof messages, so no test for caas case.